server.js

```diff
@@ -61,6 +61,48 @@ const EXTERNAL_RESOURCES = {
   search: Strings.isEmpty(Environment.RESOURCE_URI_SEARCH) ? null : Environment.RESOURCE_URI_SEARCH,
 };
 
+let exploreSlates;
+
+const fetchExploreSlates = async () => {
+  if (Environment.IS_PRODUCTION) {
+    exploreSlates = await Data.getSlatesByIds({
+      ids: [
+        //NOTE(tara): slates in prod
+        "d2861ac4-fc41-4c07-8f21-d0bf06be364c",
+        "9c2c458c-d92a-4e81-a4b6-bf6ab4607470",
+        "7f461144-0647-43d7-8294-788b37ae5979",
+        "f72c2594-b8ac-41f6-91e0-b2da6788ae23",
+        "a0d6e2f2-564d-47ed-bf56-13c42634703d",
+        "0ba92c73-92e7-4b00-900e-afae4856c9ea",
+      ],
+    });
+
+    for (let exploreSlate of exploreSlates) {
+      let user = await Data.getUserById({ id: exploreSlate.data.ownerId });
+      exploreSlate.username = user.username;
+    }
+  } else {
+    exploreSlates = await Data.getSlatesByIds({
```

Comment:
Keep in mind that this part may error for other people. Other people don't necessarily have the same slates and slateIds as you in their database. If you wanted to be safe, I would comment this out (so that you can use it to test if needed still) and return an empty array if not in production